thunder/executors/torchex.py

```diff
@@ -1883,6 +1887,21 @@ def _update_bucket_view_prim_impl(
         views[index_of_dst_view].copy_(tensor)
         return tensor
 
+    # TODO(crcrpar): Update to return None instead
```

Comment:
Why do you want to return None? If you return grad the dataflow in the trace is preserved.

Reply:
The unsharded gradients returned by the bound symbols and the trace calling this primitive will not be used thus I thought it'd be natural to return None

Reply:
Is there any safety against DCE removing this computation completely?

Reply:
I tried `DONT_DCE` with return of `None` but it didn't work so I left a comment